<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QWidget" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>863</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>863</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgba(61, 61, 61, 0.6);
color: white;
border: 1px solid rgba(204, 0, 204, 0.6);
border-radius: 5px;     
*/

enabled {
	background-color: rgba(61, 61, 61, 0.6);
	color: white;
	border: 1px solid rgba(204, 0, 204, 0.6);
	border-radius: 5px;     
}

disabled {
	background-color: rgba(61, 61, 61, 0.3);
	color: rgba(255, 255, 255,0.6);
	border: 1px solid rgba(204, 0, 204, 0.3);
	border-radius: 3px;       
}

QWidget:enabled {
	background-color: rgba(61, 61, 61, 0.6);
	color: white;
	border: 1px solid rgba(204, 0, 204, 0.6);
	border-radius: 5px;     
}

QWidget:disabled {
	background-color: rgba(61, 61, 61, 0.3);
	color: rgba(255, 255, 255,0.6);
	border: 1px solid rgba(204, 0, 204, 0.3);
	border-radius: 3px;       
}

QSpinBox{
	border: 2px solid rgba(204, 0, 204,0.6)
}

QSpinBox:disabled {
	background-color: rgba(61, 61, 61, 0.3);
	color: rgba(255, 255, 255,0.6);
	border: 1px solid rgba(204, 0, 204, 0.3);
	border-radius: 3px;       
}

QSpinBox:enabled {
	background-color: rgba(61, 61, 61, 0.6);
	color: white;
	border: 1px solid rgba(204, 0, 204, 0.6);
	border-radius: 5px;       
}

QRadioButton {
background-color: rgba(61, 61, 61, 0.6);}

QRadioButton: disabled {
background-color: rgba(61, 61, 61, 0.3);
color: rgba(255, 255, 255, 185);
border: 1px solid rgba(204, 0, 204, 0.3);
border-radius: 5px;       
}

QRadioButton: enabled {
background-color: rgba(61, 61, 61, 0.6);
color: white;
border: 1px solid rgba(204, 0, 204, 0.6);
border-radius: 5px;       
}

QLabel {
background-color: rgba(61, 61, 61, 0.6);}

QLabel: disabled {
background-color: rgba(61, 61, 61, 0.3);
color: rgba(255, 255, 255, 185);
border: 0px solid rgba(204, 0, 204,1.0)
border-radius: 5px;       
}

QLabel: enabled {
background-color: rgba(61, 61, 61, 0.6);
color: white;
border: 0px solid rgba(204, 0, 204,1.0)
border-radius: 5px;       
}
</string>
  </property>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>871</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">enabled {
	background-color: rgba(61, 61, 61, 0.6);
	color: white;
	border: 1px solid rgba(204, 0, 204, 0.6);
	border-radius: 5px;     
}</string>
   </property>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">enabled {
	background-color: rgba(61, 61, 61, 1);
	color: white;
	border: 1px solid rgba(204, 0, 204, 0.6);
	border-radius: 5px;     
}

border: 2px solid rgba(204, 0, 204,0.6)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_15">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Полнота(recall):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Прецизионность (precision):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Средняя точность(average accuracy):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>f-мера(f-measure):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="av_accurancy">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string>Затраченное время</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time_spent">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="precision">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="recall">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="f_measure">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid rgba(204, 0, 204,1.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
